Shorten.hs

```diff
@@ -0,0 +1,29 @@
+import Text.Regex.PCRE (Regex)
+import Text.Regex.Base.RegexLike
+import qualified Data.ByteString.Char8 as C (pack, unpack)
+import Network.Http.Client
+
+polrParams :: String -> String
+polrParams = (++) "http://www.polr.me/publicapi.php?action=shorten&url="
+
```

Comment:
should either url escape the param before concatenating or use a query string builder